lib/lago_http_client/lago_http_client/http_error.rb

```diff
@@ -16,6 +16,8 @@ def message
 
     def json_message
       JSON.parse(error_body)
+    rescue JSON::ParserError
+      {}
```

Comment:
Maybe we should catch the error in sentry and log it, similar to what we do there.
Similar to what we do there: [/clock.rb#L13-L16](https://github.com/getlago/lago-api/blob/aab15479d4631717f429146086905a6275dc952b/clock.rb#L13-L16)

```suggestion
    rescue JSON::ParserError => e
      Rails.logger.error("Received invalid json body: #{error_body}") # or just e.message
      Sentry.capture_exception(e)
      {}
```

Reply:
Actually, I thought we might be missing errors but here the error will be handled anyway, it's ust that we try to deserialise some json that isn't json. Sorry ignore me!